micrometer-core/src/test/java/io/micrometer/core/instrument/binder/jvm/JvmMemoryMetricsTest.java

```diff
@@ -41,45 +41,38 @@ void memoryMetrics() {
         MeterRegistry registry = new SimpleMeterRegistry();
         new JvmMemoryMetrics().bindTo(registry);
 
-        final Tag directBufferTag = Tag.of("id", "direct");
-        final Tag mappedBufferTag = Tag.of("id", "mapped");
-        assertMeters(registry.find("jvm.buffer.count").meters(), false, null, directBufferTag,
-                mappedBufferTag);
-        assertMeters(registry.find("jvm.buffer.memory.used").meters(), true, "bytes",
-                directBufferTag, mappedBufferTag);
-        assertMeters(registry.find("jvm.buffer.total.capacity").meters(), false, "bytes",
-                directBufferTag, mappedBufferTag);
+        assertJvmBufferMetrics(registry, "direct");
+        assertJvmBufferMetrics(registry, "mapped");
```

Comment:
@mweirauch  I simplified the `JvmMemoryMetricsTest` to assert pretty much the same info, but it is hopefully a little easier to read now.